### PR TITLE
test chat on web

### PR DESCRIPTION
**Pull request by rabbitmouse**
123

### CODE REVIEW

**Comment by geekdmm0xff** [comment]
xxx

**Comment by rabbitmouse** [comment]
ffff

File: ch2 [hunk 1]
```diff
@@ -1,3 +1,3 @@
 222
-
+123456
```
Comment (geekdmm0xff) [inline ch2:2]: i think 2222

Comment (rabbitmouse) [inline ch2:2]: this is my change

Comment (rabbitmouse) [reply ch2:2]: yes

**Pull request merged**